<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAjouter</class>
 <widget class="QDialog" name="DialogAjouter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>629</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Candara</family>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Nouveau article</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="1">
      <widget class="QLineEdit" name="nomArticle">
       <property name="placeholderText">
        <string>Nom de l'article</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Numéro :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="numArticle">
       <property name="placeholderText">
        <string>Numéro de l'article (format &quot;000.000.000&quot;)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Catégorie :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Designer :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="designerArticle">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Nom du designer de l'article</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Photo :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Description :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QTextEdit" name="descArticle">
       <property name="placeholderText">
        <string>Description de l'article</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Dimensions colis :</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="maximumSize">
            <size>
             <width>65</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Longueur :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="longArticle"/>
         </item>
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>cm</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="maximumSize">
            <size>
             <width>65</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Largeur :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="largArticle"/>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>cm</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="maximumSize">
            <size>
             <width>65</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Profondeur :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="profArticle"/>
         </item>
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>cm</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="maximumSize">
            <size>
             <width>65</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Poids :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="poidArticle"/>
         </item>
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>kg</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="categArt">
       <property name="placeholderText">
        <string>Catégorie de l'article</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="imageArticle">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Nom du fichier contenant la photo de l'article</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Première finition</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Couleur :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="finition">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Couleur de la finition</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Prix (€) :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="prixArticle"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Disponibilité :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="disponibilite"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>729</width>
           <height>70</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>D'autres finitions pourront être ajoutées ultérieurement depuis la modification de l'article.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pushButtonValider">
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAnnuler">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
